src/main/kotlin/app/batch/S3DirectoryWriter.kt

```diff
@@ -121,12 +122,13 @@ class S3DirectoryWriter(private val keyService: KeyService,
             // The call was transmitted successfully, but Amazon S3 couldn't process
             // it, so it returned an error response.
             e.printStackTrace()
+            logger.error("Error Writing file '$file'", e)
         } catch (e: SdkClientException) {
             // Amazon S3 couldn't be contacted for a response, or the client
             // couldn't parse the response from Amazon S3.
             e.printStackTrace()
+            logger.error("Error Writing file '$file'", e)
```

Comment:
Maybe log something different here or no point catching it separately.

Reply:
Done